sale_contact_type/readme/USAGE.rst

```diff
@@ -1,2 +1 @@
-Create/modify a children contact (Contacts > Selected or Create one > Contacts & Addresses > Press on '+ Add') and set the type to 'Ordering contact'
-Create a Sale Order (Sales > Create) and select the contact created/modified in 'Customer' field.
+Create/modify the desired children's contact (Contacts > Select or Create > Contacts & Addresses > Press 'Add' button) and set type to 'Ordering contact'.
```

Comment:
Don't remove dashes, as they are steps to do, but better put `#. ` to indicate a numbered list

Reply:
Changes done.